templates/README.md.erb

```diff
@@ -17,9 +17,17 @@ See also dockerhub tags page: https://hub.docker.com/r/fluent/fluentd-kubernetes
 
 #### Current stable
 
+As the current limitation about the number of automated builds on hub.docker.com, there are some restrictions to ship daemonset images:
+
+* `stackdriver`, `papertrail`, `syslog` images (x86_64/arm64) won't be published anymore
+* `logentries`, `loggly`, `logzio`, `s3` arm64 images won't be published anymore (x86_64 only supported)
+
+If you want to use above non published images, build it by yourself. Dockefile itself is still maintained in this repository.
+
 ##### Multi-Arch images
 <% debian.each do |image| %>
 <% path = image.split(":").first.sub("-amd64",""); tags = image.split(":").last %>
+<% next if %w(stackdriver papertrail syslog logentries loggly logzio s3).any? do |component| tags.include?(component) end %>
```

Comment:
Can we generate this definition from x86_64 and arm64 skip image name array?
I think that it is redundant to write here. Instead we should generate from skipable image name definitions.

Reply:
Fixed in https://github.com/fluent/fluentd-kubernetes-daemonset/pull/581/commits/262f82924833547bf59b33c1ebe9eebc63ebb14d.